For `table_overflow_td_align_right.html` add reference test expectation

Ahmad-S792 opened the pull request:
#### 9724eaf72f2cf7c9a2c8a8600a4fbdc4b36b0bf5
<pre>
For `table_overflow_td_align_right.html` add reference test expectation
<a href="https://bugs.webkit.org/show_bug.cgi?id=299431">https://bugs.webkit.org/show_bug.cgi?id=299431</a>

Reviewed by Antti Koivisto.

Partial Merge: <a href="https://chromium.googlesource.com/chromium/src.git/+/df66e129473c4c37c2bc5ba93ba6decbe27c6da3">https://chromium.googlesource.com/chromium/src.git/+/df66e129473c4c37c2bc5ba93ba6decbe27c6da3</a>

This changes test to be reference test, which is easier to maintain.

* LayoutTests/platform/glib/tables/mozilla_expected_failures/marvin/table_overflow_td_align_right-expected.txt: Removed.
* LayoutTests/platform/gtk/tables/mozilla_expected_failures/marvin/table_overflow_td_align_right-expected.png: Removed.
* LayoutTests/platform/ios/tables/mozilla_expected_failures/marvin/table_overflow_td_align_right-expected.txt: Removed.
* LayoutTests/platform/mac/tables/mozilla_expected_failures/marvin/table_overflow_td_align_right-expected.png: Removed.
* LayoutTests/platform/mac/tables/mozilla_expected_failures/marvin/table_overflow_td_align_right-expected.txt: Removed.
* LayoutTests/platform/win/tables/mozilla_expected_failures/marvin/table_overflow_td_align_right-expected.txt: Removed.
* LayoutTests/tables/mozilla_expected_failures/marvin/table_overflow_td_align_right-expected.html: Added.

Canonical link: <a href="https://commits.webkit.org/300511@main">https://commits.webkit.org/300511@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/3312b8d3e51a1a0ec7b9bd44e3b3f8e649c9c900

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/122784 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/42496 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/33186 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/129409 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/74888 "Built successfully") | [✅ 🛠 ios-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/1fe17b60-4a1a-45d9-bc5f-be710eb8e606) 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/43215 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/51089 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/93316 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/61953 "Passed tests") | [✅ 🛠 mac-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/c5a5a79e-b101-4766-9fda-18c3da3a728a) 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/125736 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/34452 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/109920 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/73955 "Passed tests") | | [❌ 🛠 vision-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/37c9d8f7-9e5f-4d9c-bad8-e8f4b5745b54) 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/33435 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/28076 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/72900 "Built successfully") | | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/104151 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/28289 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/132136 "Built successfully") | | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/49729 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/37861 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/101829 "Passed tests") | | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/50106 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/106136 "Passed tests") | [❌ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/101703 "Found 4 new API test failures: WebKitGTK/TestWebKitWebContext:/webkit/WebKitWebContext/no-web-process-leak, WebKitGTK/TestInspector:/webkit/WebKitWebInspector/manual-attach-detach, TestWTF:CompletionHandlerDeathTest.MainThreadHandlerOnThreadAssertsDeathTest, TestWTF:CompletionHandlerDeathTest.ConstructionThreadHandlerOnThreadAssertsDeathTest (failure)") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/25844 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/47079 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/25264 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/46499 "Built successfully") | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/49586 "Built successfully") | | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/49053 "Built successfully") | | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/52405 "Built successfully") | | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/50736 "Built successfully") | | | | 
<!--EWS-Status-Bubble-End-->

nt1m commented on LayoutTests/tables/mozilla_expected_failures/marvin/table_overflow_td_align_right-expected.html:
```diff
@@ -0,0 +1,79 @@
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+<title>testscase for table overflow handling align right</title>
+<style type="text/css">
```
you can remove `type="type/css"`

nt1m commented on LayoutTests/tables/mozilla_expected_failures/marvin/table_overflow_td_align_right-expected.html:
```diff
@@ -0,0 +1,79 @@
+   - ***** END LICENSE BLOCK ***** -->
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
```
this isn't really needed

nt1m commented on LayoutTests/tables/mozilla_expected_failures/marvin/table_overflow_td_align_right-expected.html:
```diff
@@ -0,0 +1,79 @@
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+<title>testscase for table overflow handling align right</title>
```
```suggestion
<title>Reference for table overflow handling align right</title>
```

nt1m commented on LayoutTests/tables/mozilla_expected_failures/marvin/table_overflow_td_align_right-expected.html:
```diff
@@ -0,0 +1,79 @@
+}
+#target {
+
+}
```
this can be removed

nt1m commented on LayoutTests/tables/mozilla_expected_failures/marvin/table_overflow_td_align_right-expected.html:
nit: spacing and indentation is inconsistent across this file

Ahmad-S792 replied:
It is Mozilla file.

Ahmad-S792 committed:
For `table_overflow_td_align_right.html` add reference test expectation
https://bugs.webkit.org/show_bug.cgi?id=299431

Reviewed by Antti Koivisto.

Partial Merge: https://chromium.googlesource.com/chromium/src.git/+/df66e129473c4c37c2bc5ba93ba6decbe27c6da3

This changes test to be reference test, which is easier to maintain.

* LayoutTests/platform/glib/tables/mozilla_expected_failures/marvin/table_overflow_td_align_right-expected.txt: Removed.
* LayoutTests/platform/gtk/tables/mozilla_expected_failures/marvin/table_overflow_td_align_right-expected.png: Removed.
* LayoutTests/platform/ios/tables/mozilla_expected_failures/marvin/table_overflow_td_align_right-expected.txt: Removed.
* LayoutTests/platform/mac/tables/mozilla_expected_failures/marvin/table_overflow_td_align_right-expected.png: Removed.
* LayoutTests/platform/mac/tables/mozilla_expected_failures/marvin/table_overflow_td_align_right-expected.txt: Removed.
* LayoutTests/platform/win/tables/mozilla_expected_failures/marvin/table_overflow_td_align_right-expected.txt: Removed.
* LayoutTests/tables/mozilla_expected_failures/marvin/table_overflow_td_align_right-expected.html: Added.

Canonical link: https://commits.webkit.org/300511@main
